presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchColumnHandle.java

```diff
@@ -53,16 +58,32 @@ public Type getType()
         return type;
     }
 
+    // do not serialize hidden as this is not needed on workers
+    @JsonIgnore
+    public boolean isHidden()
+    {
+        return hidden;
+    }
+
     @JsonProperty
     public boolean isSupportsPredicates()
     {
         return supportsPredicates;
     }
 
+    public ColumnMetadata getColumnMetadata()
```

Comment:
I think this is a code smell. When I wrote this (and later, when I added support for handling case-sensitive identifiers), I intentionally avoided this approach, as I don't think metadata should be derived from a ColumnHandle, but other way around. Doing it this way forces us to add spurious fields to the column handle that are not needed to identify the column or during execution (as evidenced by the need to make `hidden` non-serializable. For instance, if we have to add column comments, we'll need to store the comment in the handle -- that data has no place in the handle.

I think it'd be better to fix SHOW CREATE TABLE to not fail when a property has no metadata associated, or to allow for some form of hidden property.